molo/surveys/tests/test_adapters.py

```diff
@@ -281,33 +279,28 @@ def test_get_tag_count_only_counts_current_user(self):
         )
         MoloSurveyPageView.objects.create(
             user=self.request.user,
-            tag=self.important_tag,
             page=self.page,
         )
         MoloSurveyPageView.objects.create(
             user=another_user,
-            tag=self.important_tag,
             page=self.page,
         )
         self.assertEqual(self.adapter.get_tag_count(self.important_tag), 1)
 
     def test_get_tag_count_only_counts_specified_tag(self):
         MoloSurveyPageView.objects.create(
             user=self.request.user,
-            tag=self.important_tag,
             page=self.page,
         )
         MoloSurveyPageView.objects.create(
             user=self.request.user,
-            tag=self.another_tag,
             page=self.page,
         )
         self.assertEqual(self.adapter.get_tag_count(self.important_tag), 1)
```

Comment:
shouldn't this now return 2? since you've added two PageView objects for the same page?